### PR TITLE
new advice: use msh22 instead of msh4

### PR DESCRIPTION
**Pull request by vlukes**
This PR is inspired by #743.

### CODE REVIEW

**Comment by rc** [comment]
Thanks! What about adding also an information about why it is a good idea to use a particular gmsh format?

File: doc/faq.rst [hunk 1]
```diff
@@ -128,6 +128,10 @@ Mesh-Related Tasks
    ``<dimension>`` is the topological dimension of cells that should be in the
    mesh. For example, ``-d 2`` stores only the 2D cells.
 
+#. Use ``msh22`` format instead of the default ``msh4`` when generating a mesh 
+   with ``gmsh``::
```
Comment (rc) [inline doc/faq.rst:132]: (the ``msh4`` format does not work well with the -2 option)??

Comment (vlukes) [reply doc/faq.rst:132]: OK, I would write "It is suggested to use ..., msh22 is more reliable and foolproof."

Comment (rc) [reply doc/faq.rst:132]: OK!

Comment (vlukes) [reply doc/faq.rst:132]: Updated.

Comment (rc) [reply doc/faq.rst:132]: LGTM, just fix the line length, please :)

Comment (vlukes) [reply doc/faq.rst]: Fixed :)

**Pull request merged**